docs/participate/dex/04-Core Concepts/02-core_concepts_2.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Financials, Fees, APY
+sidebar_label: Financials, Fees, APY
```

Comment:
APR instead of APY

Reply:
You are right, it is APR, 

FYI: I will update the Financials, Fees, APR soon with more preciseness to it soon.